pkg/controller/pod.go

```diff
@@ -1028,21 +1038,30 @@ func (c *Controller) handleDeletePod(key string) error {
 			return err
 		}
 		for _, port := range ports {
-			klog.Infof("clean migrate options for vm lsp %s", port.Name)
```

Comment:
Why is this log removed?

Reply:
there is only one usage of this function. and also, there is a log in this function.

if not removed, there are two logs. which seems redundantly.